docs/sphinx/source/whatsnew/v0.9.5.rst

```diff
@@ -19,6 +19,11 @@ Enhancements
 * Added the optional `string_factor` parameter to
   :py:func:`pvlib.snow.loss_townsend` (:issue:`1636`, :pull:`1653`)
 
+* Added `haydavies` sky-diffuse transposition model to
+  :py:func:`pvlib.bifacial.infinite_sheds.get_irradiance` and
+  :py:func:`pvlib.bifacial.infinite_sheds.get_irradiance_poa`
+  (:pull:`1668`)
```

Comment:
```suggestion
* Added an optional ``model`` parameter to
  :py:func:`pvlib.bifacial.infinite_sheds.get_irradiance` and
  :py:func:`pvlib.bifacial.infinite_sheds.get_irradiance_poa`
  to enable use of the hay-davies sky diffuse irradiance model
  instead of the default isotropic model. (:pull:`1668`)
```